pylint_odoo/test/main.py

```diff
@@ -9,6 +9,7 @@
 
 
 EXPECTED_ERRORS = 52
+EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
```

Comment:
@pedrobaeza What do you propose to avoid having this after couple contributions?
```
EXPECTED_ERRORS = 52
EXPECTED_ERRORS += 2  # Errors due to C8106: Wrong version format
EXPECTED_ERRORS += 3  # Errors due to XXXX
EXPECTED_ERRORS += 4  # Errors due to YYYY
EXPECTED_ERRORS += 5  # Errors due to ZZZZ
```